File: app/src/main/java/org/michaelbel/moviemade/ui/SettingsActivity.java
package org.michaelbel.moviemade.ui;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.Nullable;

import org.michaelbel.moviemade.R;
import org.michaelbel.moviemade.databinding.ActivitySettingsBinding;
import org.michaelbel.moviemade.mvp.base.BaseActivity;
import org.michaelbel.moviemade.ui.fragment.SettingsFragment;

public class SettingsActivity extends BaseActivity {

    public ActivitySettingsBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_settings);

        setSupportActionBar(binding.toolbar);
        startFragment(new SettingsFragment(), binding.fragmentView);
    }
}

File: app/src/main/java/org/michaelbel/moviemade/presentation/features/settings/SettingsActivity.kt
package org.michaelbel.moviemade.presentation.features.settings

import android.os.Bundle
import androidx.fragment.app.transaction
import kotlinx.android.synthetic.main.activity_settings.*
import org.michaelbel.moviemade.R
import org.michaelbel.moviemade.presentation.common.base.BaseActivity

class SettingsActivity: BaseActivity(R.layout.activity_settings) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setSupportActionBar(toolbar)
        toolbar.setNavigationOnClickListener { finish() }
        supportActionBar?.setTitle(R.string.settings)

        if (savedInstanceState == null) {
            supportFragmentManager.transaction {
                replace(container.id, SettingsFragment.newInstance())
            }
        }
    }
}